Data-Structure-ARRAYS.md

```diff
@@ -0,0 +1,61 @@
+#Arrays
```

Comment:
```diff
-#Arrays
+# Data Structure Arrays
```

Don't use `:` on the first title, they break the generator and get removed anyways.